src/test/groovy/net/rubygrapefruit/platform/file/AbstractFileEventsTest.groovy

```diff
@@ -382,12 +375,12 @@ abstract class AbstractFileEventsTest extends Specification {
             watcher.startWatching(rootDir)
             readyLatch.await()
             startModifyingLatch.countDown()
-            Thread.sleep(200)
+            Thread.sleep(500)
             watcher.close()
+            assert uncaughtFailureOnThread.empty
```

Comment:
This is now asserted thrice, why?

Reply:
I am asserting it here, so the test ends early when the first error occurs. I removed the other assertion.